calculator/src/app/page.tsx

```diff
@@ -0,0 +1,37 @@
+'use client'
+
+import useCalculator from "./useCalculator";
+
+export default function Home() {
+	const {
+    formula,
+    number,
+    add,
+    subtract,
+    multiply,
+    divide,
+    calculate,
+    clearAll
+  } = useCalculator()
+
+  return (
+		<>
+			<div className="calculator">
+				<h1 id="total">{formula}</h1>
+				<div className="digits flex">
+					{Array.from({ length: 10 }, (_, i) => (<button key={i} className="digit" onClick={()=>(number(9-i))}>{9-i}</button>))}
```

Comment:
- 숫자 배열은 컴포넌트 외부에서 상수로 선언해도 무방해보입니다.
- 오름차순 배열을 내림차순으로 바꾸기 위해 index를 빼는 것보다 [Array.prototype.toReversed()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/toReversed)나 [Array.prototype.toSorted()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/toSorted)와 같은 메서드를 체이닝하여 사용하는 것이 코드를 더 명확하게 만들고 다른 개발자들도 코드를 더 쉽게 이해할 수 있습니다. flexbox의 flow-direction이나 grid를 사용하여 css로 뷰로직을 처리하는 것도 고려해보세요.
- JSX 마크업 내에서 값을 생성하는 것보다 배열을 그대로 렌더링하는 것이 뷰 로직의 가독성 향상에 도움이 됩니다.

```suggestion
const NUMBERS = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9];
return (
  <ul>
    {NUMBERS.map((value) => (
      <li>
        <button key={value}>{value}</Button>
      </li>
    ))}
  </ul>
)
```